app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt

```diff
@@ -24,10 +25,18 @@ class MeetingRequestService : FirebaseMessagingService() {
    */
   override fun onMessageReceived(remoteMessage: RemoteMessage) {
     super.onMessageReceived(remoteMessage)
-    remoteMessage.notification?.let { notification ->
-      val title = notification.title ?: "Default Title"
-      val body = notification.body ?: "Default Body"
-      showNotification(title, body)
+    Log.d("MEETING REQUEST", "received stuff")
+    val senderUid = remoteMessage.data["senderUID"] ?: "null"
+    val message = remoteMessage.data["message"] ?: "null"
+    val title = remoteMessage.data["title"]
```

Comment:
Nitpick: Even if the == operator is null-safe, adding here a null check for consistency would be better